sdk/documentintelligence/Azure.AI.DocumentIntelligence/tests/DocumentIntelligenceAdministrationClient/TrainingOperationLiveTests.cs

```diff
@@ -12,14 +12,37 @@ namespace Azure.AI.DocumentIntelligence.Tests
 {
     public class TrainingOperationLiveTests : DocumentIntelligenceLiveTestBase
     {
+        // Example: 3eb2e5b5-b9d3-4b5a-ac31-90d945f4b4e4
+        private const string AnalysisOperationIdPattern = @"^[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}$";
+
         // Example: 31466498286_3eb2e5b5-b9d3-4b5a-ac31-90d945f4b4e4
-        private const string OperationIdPattern = @"^[0-9]+_[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}$";
+        private const string TrainingOperationIdPattern = @"^[0-9]+_[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}$";
```

Comment:
We now have two different ID patterns we want to test depending on the type of operation we have.